contributors/guide/pull-requests.md

```diff
@@ -590,7 +597,33 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
-## Fixing linter issues
+## Large or Automatic Edits
+
+Some tools make it very easy to create large Pull Requests, for example:
+- global search/replace
+- linters which automatically correct issues (see also next section)
+- large language models (LLMs) which generate code or documentation
+
+To make it easier for reviewers to handle such Pull Requests, please explain
+how it was generated in the "Special notes for your reviewer" section of the
```

Comment:
Should we add some suggestions/questions in current Pull Request template for this part?

Reply:
Hesitant about that. The template is pretty long

Reply:
I agree our current template is already long. This is more for contributors to use when they are closing PRs, so they don't have to repeat the same thing over and over again, but rather just use a "saved reply" feature from GH.